assets/messages.yml

```diff
@@ -209,6 +209,23 @@ messages:
         The authentication services are offline.
         This will be fixed shortly, please hang tight. Check tweets from [@MojangSupport|https://twitter.com/MojangSupport] for the current status.
 
+        %quick_links%
+      fillname: []
+  banned:
+    - project:
+        - mcd
+        - mcpe
+        - realms
+        - web
+      name: Banned
+      shortcut: ban
+      message: |-
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
+        Additionally bans are permanent and non-negotiable. Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/360052618531-Why-have-I-been-banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
```

Comment:
Perhaps add a comma after "Additionally"?

Reply:
Definitely needs a comma there. Otherwise looks fine.

Reply:
Do we really need to have it say "Additionally bans are permanent and non-negotiable." on the bug tracker? Directing them to the help article should be enough. It's not really the bug tracker's job to convey that information. Also, what happens if Microsoft ever changes the policy to add an appeal process? Then all those comments are now incorrect.

Reply:
I edited out the statement about bans being permanent.